### PR TITLE
Feature quotes

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: archive-luminaries.php [hunk 1]
```diff
@@ -42,9 +42,22 @@
...
-  <?php get_template_part('partials/pagination'); ?>
+    <?php
+      $quote_text = IGV_get_option('_igv_quote_options', '_igv_luminaries_quote_text');
+      $quote_person = IGV_get_option('_igv_quote_options', '_igv_luminaries_quote_person');
+      $quote_luminary = IGV_get_option('_igv_quote_options', '_igv_luminaries_quote_luminary');
+
+      if ($quote_text && $quote_person) {
+    ?>
+    <div class="grid-row margin-top-large">
+      <?php render_quote($quote_text, $quote_person, $quote_luminary); ?>
```
Comment (mikeyrayvon) [inline archive-luminaries.php]: why is this not a partial tho?

File: lib/functions-custom.php [hunk 2]
```diff
@@ -49,4 +49,34 @@ function menuActiveClasses($item = null, $type = null, $classes = null) {
 
 function render_embed_caption($caption) {
   echo '<h5 class="text-align-center margin-top-tiny margin-bottom-tiny">' . $caption . '</h5>';
+}
+
+
+function render_quote($text = null, $person = null, $luminary = null) {
```
Comment (mikeyrayvon) [inline lib/functions-custom.php]: these vars are coming from site options. easily could be a partial instead of a function

Comment (mikeyrayvon) [inline lib/functions-custom.php]: or is there another use case that doesnt use theme options values?

Comment (varffs) [inline lib/functions-custom.php]: i dont know if there will be another context not using theme options. But how will you tell the partial the context? There are 3 different groups theme options here for each page. One is a post type archive template the other to custom pages

Comment (mikeyrayvon) [inline lib/functions-custom.php]: alright i'll merge this in interest of time 👍

**Pull request merged**